src/snowflake/snowpark/modin/pandas/dataframe.py

```diff
@@ -158,6 +159,11 @@ def __init__(
 
         self._siblings = []
 
+        if not ast_stmt:
```

Comment:
In column.py we're using if ... is None; we should use a consistent pattern for this check – whatever is the canonical approach in Python. It seems that `is None` makes more sense?